README.md

```diff
@@ -47,7 +47,7 @@ assert_eq!(arena.get(foo), None);
 | `size_of::<Option<Index>>()` | 8           | 24                 | 8       | 16   |
 | Max Elements                 | 2³²         | 2⁶⁴                | 2³²     | 2⁶⁴  |
 | Non-`Copy` Values            | Yes         | Yes                | Yes     | Yes  |
-| `no_std` Support             | No          | Yes                | Yes     | No   |
+| `no_std` Support             | Yes         | Yes                | Yes     | No   |
```

Comment:
Can you update the table and MSRV in `lib.rs` as well? This crate's README is generated by editing that file, and then running [cargo-readme](https://github.com/livioribeiro/cargo-readme).